### PR TITLE
Various small notebook fixes

### PR DESCRIPTION
**Pull request by katalinic-gc**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: notebooks/introduction_to_optimum_graphcore.ipynb [hunk 1]
```diff
@@ -405,7 +405,7 @@
     "\n",
     "> We also call the `.half()` method to cast all the model weights to half-precision (FP16). The `.train()` sets the PyTorch model to training mode.\n",
     "\n",
-    "> If you unfamiliar with training in half precision on IPU, then we have a tutorial on [Half and Mixed Precision in Poptorch](https://github.com/graphcore/tutorials/tree/master/tutorials/pytorch/tut3_mixed_precision)."
+    "> If you are unfamiliar with training in half precision on IPU, then we have a tutorial on [Half and Mixed Precision in Poptorch](https://github.com/graphcore/tutorials/tree/sdk-release-3.0/tutorials/pytorch/mixed_precision)."
```
Comment (payoto) [inline notebooks/introduction_to_optimum_graphcore.ipynb:408]: Better to point at master:

```suggestion
    "> If you are unfamiliar with training in half precision on IPU, then we have a tutorial on [Half and Mixed Precision in Poptorch](https://github.com/graphcore/tutorials/tree/master/tutorials/pytorch/mixed_precision)."
```

File: notebooks/introduction_to_optimum_graphcore.ipynb [hunk 2]
```diff
@@ -88,7 +88,7 @@
     "4. Connect via SSH to your remote machine, forwarding your chosen port:\n",
     "`ssh -NL <port number>:localhost:<port number> <your username>@<remote machine>`\n",
     "\n",
-    "For more details about this process, or if you need troubleshooting, see our [guide on using IPUs from Jupyter notebooks](../../standard_tools/using_jupyter/README.md)."
+    "For more details about this process, or if you need troubleshooting, see our [guide on using IPUs from Jupyter notebooks](https://github.com/graphcore/tutorials/tree/sdk-release-3.0/tutorials/standard_tools/using_jupyter)."
```
Comment (payoto) [inline notebooks/introduction_to_optimum_graphcore.ipynb:91]:
```suggestion
    "For more details about this process, or if you need troubleshooting, see our [guide on using IPUs from Jupyter notebooks](https://github.com/graphcore/tutorials/tree/master/tutorials/standard_tools/using_jupyter)."
```

File: notebooks/language_modelling_from_scratch.ipynb [hunk 3]
```diff
@@ -291,7 +291,7 @@
...
     "\n",
-    "In this notebook, we are using both data parallelism and pipeline parallelism (see this [tutorial](https://github.com/graphcore/tutorials/tree/master/tutorials/pytorch/tut2_efficient_data_loading) for more). Therefore the global batch size, which is the actual number of samples used for the weight update, is determined with three factors:\n",
+    "In this notebook, we are using both data parallelism and pipeline parallelism (see this [tutorial](https://github.com/graphcore/tutorials/tree/sdk-release-3.0/tutorials/pytorch/efficient_data_loading) for more). Therefore the global batch size, which is the actual number of samples used for the weight update, is determined with three factors:\n",
```
Comment (payoto) [inline notebooks/language_modelling_from_scratch.ipynb:294]:
```suggestion
    "In this notebook, we are using both data parallelism and pipeline parallelism (see this [tutorial](https://github.com/graphcore/tutorials/tree/master/tutorials/pytorch/efficient_data_loading) for more). Therefore the global batch size, which is the actual number of samples used for the weight update, is determined with three factors:\n",
```

File: notebooks/multiple_choice.ipynb [hunk 4]
```diff
@@ -119,7 +119,7 @@
     "\n",
     "This notebook is built to run  with any model checkpoint from the [Model Hub](https://huggingface.co/models) as long as that model has a version with a mutiple choice head and is supported by Optimum Graphcore. The IPU config files of the supported models are available in Graphcore's [Hugging Face account](https://huggingface.co/Graphcore). You can also create your own IPU config file locally. \n",
     "\n",
-    "In this notebook, we are using both data parallelism and pipeline parallelism (see this [tutorial](https://github.com/graphcore/tutorials/tree/master/tutorials/pytorch/tut2_efficient_data_loading) for more). Therefore the global batch size, which is the actual number of samples used for the weight update, is determined with three factors:\n",
+    "In this notebook, we are using both data parallelism and pipeline parallelism (see this [tutorial](https://github.com/graphcore/tutorials/tree/sdk-release-3.0/tutorials/pytorch/efficient_data_loading) for more). Therefore the global batch size, which is the actual number of samples used for the weight update, is determined with three factors:\n",
```
Comment (payoto) [inline notebooks/multiple_choice.ipynb:122]:
```suggestion
    "In this notebook, we are using both data parallelism and pipeline parallelism (see this [tutorial](https://github.com/graphcore/tutorials/tree/master/tutorials/pytorch/efficient_data_loading) for more). Therefore the global batch size, which is the actual number of samples used for the weight update, is determined with three factors:\n",
```

**Review by payoto — COMMENTED** [review]
The changes you have back ported look good, although can you avoid pointing at the `sdk-release-3.0` branch of tutorials instead can you point to `master`:

Change all of those: `https://github.com/graphcore/tutorials/tree/sdk-release-3.0`

to `https://github.com/graphcore/tutorials/tree/master`

File: notebooks/multiple_choice.ipynb (hunk 4 above)
Comment (katalinic-gc) [reply notebooks/multiple_choice.ipynb]: Search and replaced them all

**Pull request merged**